src/pykiso/test_coordinator/test_execution.py

```diff
@@ -214,6 +235,7 @@ def execute(
     step_report: Optional[Path] = None,
     pattern_inject: Optional[str] = None,
```

Comment:
what does the pattern inject do? I mean different then your tc_filter functionality?

Reply:
It filters the test files / Test modules (py files).
Its currently used to override the test_filter_pattern in the yaml file.
test_filter_pattern: '*.py'